onedal/datatypes/table.cpp

```diff
@@ -17,12 +17,12 @@
 #include "oneapi/dal/table/common.hpp"
 #include "oneapi/dal/table/homogen.hpp"
 
-#ifdef ONEDAL_DPCTL_INTEGRATION
-#include "onedal/datatypes/data_conversion_dpctl.hpp"
-#endif // ONEDAL_DPCTL_INTEGRATION
+#ifdef ONEDAL_DATA_PARALLEL
```

Comment:
Maybe my understanding is incorrect, but shouldn't this change from `ONEDAL_DPCTL_INTEGRATION` to `ONEDAL_DATA_PARALLEL` impact the build process?
Should some infra scripts or build instruction be updated to reflect this change?

Reply:
Good question.
DPCTL already removed as a build dependency, and the build scripts updated there.
Please see https://github.com/intel/scikit-learn-intelex/pull/1685